.gitignore

```diff
@@ -13,7 +13,7 @@
 
 # misc
 .DS_Store
-.env.local
+_spanish_soup_env.local
```

Comment:
øh ?

Reply:
Remove... .env.local renamed not to be included in build